src/pages/home/report/ReportActionsView.js

```diff
@@ -145,7 +145,7 @@ class ReportActionsView extends React.Component {
             this.updateNewMarkerAndMarkReadOnce();
         }
 
-        this.fetchData();
+        Report.openReport(this.props.reportID);
```

Comment:
All of the `updateNewMarkerAndMarkReadOnce()` stuff can go now. That was only existing to support a race condition where the `maxSequenceNumber` is not up to date because the report object had not loaded yet. Since we are relying on the report actions in the backend we can get rid of it.

Reply:
removed all of them